scenarios/search-menu.json

```diff
@@ -38,6 +38,26 @@
       "hideSelectors": ["#mainTabContainer"],
       "onReadyScript": "search/full-text-result.js"
     },
+    {
+      "label": "Full text Search - results - Contact",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Contact&text="
+    },
+    {
+      "label": "Full text Search - results - Activities",
+      "url": "{url}/civicrm/contact/search/custom?csid=15&reset=1&force=1&table=Activity&text="
```

Comment:
what is this `cid` param? Is it safe to hardcode its value?

Reply:
*csid* is already provided in the URLs for previously written scenarios. And if we don't provide that in the URL the page shows error . Also it gets automatically generated when user clicks on Full text search option from the main navigation.